src/Microsoft.Performance.SDK.Tests/ProcessingOptionsResolverTests.cs

```diff
@@ -0,0 +1,303 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.SDK.Processing;
+using Microsoft.Performance.SDK.Processing.DataSourceGrouping;
+using Microsoft.Performance.SDK.Processing.Options;
+using Microsoft.Performance.SDK.Runtime;
+using Microsoft.Performance.Testing;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source123;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source4;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source5;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Performance.SDK.Tests
+{
+    [TestClass]
+    public class ProcessingOptionsResolverTests
+    {
+        [TestMethod]
+        [IntegrationTest]
+        public void DefaultGlobalProcessingOptionsResolverTest()
+        {
+            var sut = GlobalProcessingOptionsResolver.Default;
```

Comment:
This test can be dramatically simplified by removing the `GlobalProcessingOptionsResolver.Default` static field and just having the `EngineCreateInfo` ctor do `new GlobalProcessingOptionsResolver(ProcessorOptions.Default)`. Then, the only test you need for the entire class is "Create a GlobalProcessingOptionsResolver with a specified option. Assert that option is always returned when GetProcessorOptions is called".

Reply:
If it's explicitly using the new `GlobalProcessingOptionsResolver(ProcessorOptions.Default)` or not I think this redundant test is good to have since it'll be a common code path.